src/core/models/components.ts

```diff
@@ -223,6 +224,17 @@ const components = createModel({
           parent: payload.parentName,
           rootParentType: payload.rootParentType || payload.type,
         }
+        if (ComponentWithRefs.includes(payload.type)) {
+          const ref = `${id.replace('-', '_')}_ref`.toLowerCase()
+          draftState.components['root'].params?.push({
+            name: ref,
+            type: 'any',
```

Comment:
use exact types

Reply:
There are multiple types for refs. Should we maintain a dictionary for that?

Reply:
yes

Reply:
done!